book/src/developer-guides/frame-nft.md

```diff
@@ -0,0 +1,52 @@
+# Overview
+Frame-NFT is aimed at developers to provide them with a reliable and adjustable codebase for NFT-based projects. 
+This is especially necessary due to the various forms that NFTs can present as, each with their own unique traits.
+In the following we will do a quick recap of what makes NFTs special followed by how that is reflected in Frame-NFT's code.
+And finally, we will discuss two use cases.
+
+---
+
+# Recap
+NFTs (Non-Fungible-Tokens) have a few characteristics setting them apart from fungible ones like currency, precious metals, or carbon credits:
+* Non - Interchangeable 
+* Specific creator and owner
+* Unique digital signature
+* Unique asset like Art, Music or Property
+* Provide complete history of their creation and trades
```

Comment:
Technically regular currencies do too (in blockchain context). `Currency` is a very common term in Substrate, so anything token/currency related is assumed to be chain related

Reply:
Alrighty, i think this one might lead  to wrong expectations, will cut.